exercises/mark-griffith/answers.js

```diff
@@ -0,0 +1,135 @@
+// Question 1
+function greet(greet) {
```

Comment:
Change this. You don't want to get in the habit of naming your parameters the same thing as your functions. You may end up calling or referencing a function when you really want to reference the parameter. A better parameter name would be `name`.